core/src/main/scripts/importer/validateData.py

```diff
@@ -1020,11 +1017,40 @@ def parseFeatureColumns(self, nonsample_col_vals):
         return self.checkGeneIdentification(hugo_symbol, entrez_id)
 
 
-class CNAValidator(GenewiseFileValidator):
+class ContinuousValuesValidator(GenewiseFileValidator):
+    """Validator for matrix files mapping floats to gene/sample combinations.
+
+    Allowing missing values indicated by GenewiseFileValidator.NULL_VALUES.
+    """
+
+    def checkValue(self, value, col_index):
+        """Check a value in a sample column."""
+
+        stripped_value = value.strip()
+        if stripped_value not in self.NULL_VALUES and not self.checkFloat(stripped_value):
+            self.logger.error("Value is neither a real number nor " + ', '.join(self.NULL_VALUES),
+                              extra={'line_number': self.line_number,
+                                     'column_number': col_index + 1,
+                                     'cause': value})
 
-    """Sub-class CNA validator."""
+
+class CNAValidator(GenewiseFileValidator):
+    """Sub-class CNA validator.
+    """
 
     OPTIONAL_HEADERS = ['Cytoband'] + GenewiseFileValidator.OPTIONAL_HEADERS
+
+    def checkGeneIdentification(self, hugo_symbol, entrez_id):
```

Comment:
Credits go to @fedde-s !

Reply:
Now I only hope that the lack of default argument values in this override isn't a problem--code written to interact with `Validator().checkGeneIdentification()` may expect it to work with fewer than two arguments. E.g. `self.checkGeneIdentification(entrez_id=spam)`.

Reply:
Unless there's a specific reason to raise an exception in those cases, it might be useful to match the `=None` from the method you're extending. Using `*args` and `**kwargs` to deduplicate that completely is probably overkill here.